### ISSUE TITLE
Event `Remarked` does not appear in events section of the explorer

### ISSUE THREAD

**Issue by Imod7** [polkadot-js/apps#8777]
* **I'm submitting a ...**

  - [x] Bug report
  - [ ] Feature request
  - [ ] Support request
  - [ ] Other

* **What is the current behavior and expected behavior?**

I built and run a Substrate node using the [node template](https://github.com/substrate-developer-hub/substrate-node-template) and then I open pjs apps locally (connected to Development).
I noticed that when I execute the extrinsic `remarkWithEvent` (under `system`) in the [Extrinsics section](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/extrinsics) it is correctly submitted & included in a block. I can confirm that because I see the extrinsic if I click on the last or previous to last block (as soon as I submit the extrinsic). 

<img width="900" alt="image" src="https://user-images.githubusercontent.com/12569221/212104498-7ed4e844-e90c-45d7-b56f-080bff1d0a41.png">

However, in the [Explorer](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2F127.0.0.1%3A9944#/explorer) in the `recent events` section I do not see anything. 

<img width="1253" alt="image" src="https://user-images.githubusercontent.com/12569221/212104722-b4de864d-4b92-43ab-b8db-62b4658f3380.png">

I would expect the [Remarked event](https://github.com/paritytech/substrate/blob/master/frame/system/src/lib.rs#L497) to appear also there.

* **What is the motivation for changing the behavior?**

Since the events from other extrinsics are printed in this section, I think it would make sense to print also this event so it aligns with the general functionality of the tool.

* **Please tell us about your environment:**

Mac OS, Firefox browser

  - Version: 
  - Environment:

    - [ ] Node.js
    - [x] Browser
    - [ ] Other (limited support for other environments)

  - Language:

    - [ ] JavaScript
    - [ ] TypeScript (include tsc --version)
    - [ ] Other

**Comment by jacogr** [comment]
This is by design - all system events are filtered. (Along with a number of other events)

**Comment by jacogr** [comment]
If we add all the block events, it is way too noisy and useful events drop off.

There could certainly be some useful events that can/should be added (and some non useful events that can/should be filtered). So always happy for PRs to extend the list found here - https://github.com/polkadot-js/apps/blob/d34424a59397a5ae2f1d2cd318fe313e37cc3c3b/packages/react-hooks/src/ctx/BlockEvents.tsx#L32-L52

TL;DR the explorer view is meant to be filtered to try and get to useful info, it may not be perfect. (But is is certainly not "a bug" that it doesn't show everything, it intentional)
